coins/exchange_rate_test.go

```diff
@@ -13,41 +13,34 @@ import (
 
 func TestExchangeRate_ToXMR(t *testing.T) {
 	rate := StrToExchangeRate("0.25") // 4 XMR * 0.25 = 1 ETH
-	ethAmount := StrToDecimal("1")
+	ethAssetAmt := StrToETHAsset("1", nil)
 	const expectedXMRAmount = "4"
-	xmrAmount, err := rate.ToXMR(ethAmount)
+	xmrAmount, err := rate.ToXMR(ethAssetAmt)
 	require.NoError(t, err)
 	assert.Equal(t, expectedXMRAmount, xmrAmount.String())
 }
 
-func TestExchangeRate_ToXMR_roundDown(t *testing.T) {
+func TestExchangeRate_ToXMR_exceedsXMRPrecision(t *testing.T) {
+	// 3.1/0.333333 calculated to 13 decimals is 9.3000093000093 (300009 repeats indefinitely)
 	rate := StrToExchangeRate("0.333333")
-	ethAmount := StrToDecimal("3.1")
+	ethAssetAmt := StrToETHAsset("3.1", nil)
 
-	// 3.1/0.333333 calculated to 13 decimals is 9.3000093000093 (300009 repeats indefinitely)
-	// This calculator goes to 200 decimals: https://www.mathsisfun.com/calculator-precision.html
-	// XMR rounds at 12 decimal places to:
-	const expectedXMRAmount = "9.300009300009"
+	_, err := rate.ToXMR(ethAssetAmt)
+	expectedErr := "3.1 ETH / 0.333333 exceeds XMR's 12 decimal precision, try 3.099999999999899997"
```

Comment:
am I wrong or does 3.1/0.333333 have only 11 decimal places? i appears to be 9.30000930001